cyvlfeat/fisher/cyfisher.pyx

```diff
@@ -20,34 +20,57 @@ from cyvlfeat._vl.fisher cimport VL_FISHER_FLAG_FAST
 
 
 @cython.boundscheck(False)
-cpdef cy_fisher(float[:, :] X,
-                float[:, :] MEANS,
-                float[:, :] COVARIANCES,
-                float[:] PRIORS,
+cpdef cy_fisher(np.ndarray X,
```

Comment:
ping @AlexisMignon - My [PR is here](https://github.com/alexismignon/cyvlfeat/pull/1) that gives my suggestion on how to unify this.